fits_schema/header.py

```diff
@@ -49,7 +50,7 @@ class HeaderCard:
         match str values case insensitively
     '''
     def __init__(
-        self, keyword=None, *, required=True,
+        self, keyword=None, description: str = "", *,required=True,
```

Comment:
Any particular reason why making the description a pos arg?

Reply:
No, was a mistake when I was trying some things. I think later, I will refactor so that description is the positional one, and keyword is not, but for now I'll keep it as is.